npc/re/merchants/Extended_Ammunition.txt

```diff
@@ -0,0 +1,479 @@
+		close2;
+		callshop "Selling_Items", 2;
+		end;
+	case 5:
+		mes "[Ammunition Merchant]";
+		mes "You can take a look at other things more calmly.";
+		close;
+	}
+	end;
+	
+OnInit:
+    setunitdata getnpcid(0),UNPC_GROUP_ID,13;	
+	end;
+}
+
+//= Ninja Tool Merchant
+//============================================================
+-	script	#over_kunai	HIDDEN_NPC,{
+	mes "[Ninja Tool Merchant]";
+	mes "Hi, what can I do to help you?";
+	next;
+	switch ( select( "Buy Ninjutsu Tool", "Parchment Exchange", "Sell Items", "Cancel" )) {
+	case 1:
+		close2;
+		callshop "Weapons_Ninja", 1;
+		end;
+	case 2:
+		close2;	
+		callshop "Box_Exchange_Ninja", 1;
+		end;
+	case 3:
+		close2;	
+		callshop "Selling_Items", 2;
+		end;
+	case 4:
+		end;
+	}
+	end;
+	
+OnInit:
+    setunitdata getnpcid(0),UNPC_GROUP_ID,14;
+	end;
+}
+
+//= Mado Gear Merchant
+//============================================================
+-	script	#over_madogear	HIDDEN_NPC,{
+	mes "[Mado Gear Merchant]";
+	mes "Hi!";
+	mes "Here you will find equipment for Madogear, except those that cannot be sold!";
+	mes "How can I help you?";
+	next;
+	switch ( select( "Buy Component", "Consumables and Cooling Device", "Box Exchange", "Sell Items", "Cancel" )) {
+	case 1:
+		mes "[Mado Gear Merchant]";
+		mes "I will start preparing the Madogear pieces.";
+		close2;
+		callshop "Weapons_Mechanic", 1;
+		end;
+	case 2:
+		mes "[Mado Gear Merchant]";
+		mes "I will start preparing the consumables used by Madogear.";
+		close2;
+		callshop "Ammunition_Mechanic", 1;
+		end;
+	case 3:
+		mes "[Mado Gear Merchant]";
+		mes "I will arrange for the consumable packs used by MadoGear to be boxed!";
+		close2;
+		callshop "Box_Exchange_Mechanic", 1;
+		end;
+	case 4:
+		mes "[Mado Gear Merchant]";
+		mes "Do you have something to sell?";
+		close2;
+		callshop "Selling_Items", 2;
+		end;
+	case 5:
+		mes "[Mado Gear Merchant]";
+		mes "You can take a look at other things more calmly.";
+		close;
+	}
+	end;
+	
+OnInit:
+    setunitdata getnpcid(0),UNPC_GROUP_ID,11;
+	end;
+}
+
+//= Talisman Merchant
+//============================================================
+-	script	#over_talisman	HIDDEN_NPC,{
+	mes "[Talisman Merchant]";
+	mes "Hello. Did you come to see the spirits?";
+	next;
+	switch ( select( "Buy Talisman", "Talisman Bundle Exchange", "Cancel" )) {
+	case 1:
+		mes "[Talisman Merchant]";
+		mes "You'll like it.";
+		close2;
+		callshop "Talisman_Merchant", 1;
+		end;
+	case 2:
+		mes "[Talisman Merchant]";
+		mes "We also exchange for bundles.";
+		close2;
+		callshop "Talisman_Bundle", 1;
+		end;
+	case 3:
+		mes "[Talisman Merchant]";
+		mes "Thank you. Come again.";
+		close;
+	}
+	end;
+	
+OnInit:
+    setunitdata getnpcid(0),UNPC_GROUP_ID,12;
+	end;
+}
+
+//= Basic Shops
+//============================================================
+
+//= Archer
+-	shop	Arrow_Archer	HIDDEN_NPC,1750:-1:1,1770:-1:1,1751:-1:1,1752:-1:1,1754:-1:1,1755:-1:1,1756:-1:1,1757:-1:1,1762:-1:1,1065:-1:1,7940:-1:1
```

Comment:
The definition is wrong and must be updated for the normal shop
normal: `<itemid>:<price>`
market: `<itemid>:<price>:<quantity>`

Btw what is the difference between an unlimited marketshop and a normal shop?

Reply:
I forgot about it! Yesterday I posted this at bedtime. Very tired. It turned out that I wrote without checking.